### ISSUE TITLE
Warn in Read Me that inverting fn keys not possible anymore

### ISSUE THREAD

**Issue by martzell** [ben-z/free-sidecar#56]
The checkbox "Use F1, F2, etc. keys as standard function keys" in System Preferences » Keyboard is replaced by Touch Bar options but my MacBook Pro has no Touch Bar. defaults write experiments I've tried did not work.

**Workaround:** Fluor App. Even better: enables fn invert only for selected apps.

**Mac model:** MacBook Pro (Retina, 15-inch, Mid 2015)
**Mac version** (optional): 10.15.3 (19D76)

**Comment by ben-z** [comment]
Oh no I missed this. Thanks for suggesting! I will update the README later.

A built-in solution may be (update: @martzell said this doesn't work. I don't have an access to a device to test this. If this doesn't work for you, use 3rd party apps such as Fluor instead):

1. To use function keys as F1, F2, etc.
```
defaults write -g com.apple.keyboard.fnState -bool true
```
2. To use function keys as media keys
```
defaults write -g com.apple.keyboard.fnState -bool false
```
Make sure to log out and log back in for the changes to take effect.

**Comment by martzell** [comment]
defaults write does not work. But Fluor is a great tool too have F1… keys on a per app basis.

And my keyboard system preference pane reverted to normal non-touchbar style, don't know why. No OS X system software update involved.

**Comment by ben-z** [comment]
Closed by 39ddb49b499402ede3019242a6a4c876ba8895be

**Comment by Petingo** [comment]
I met the same problem and [Karabiner-Elements](https://karabiner-elements.pqrs.org/) solved it. If Fluor doesn't work, try this :)
